prosody/rootfs/defaults/conf.d/jitsi-meet.cfg.lua

```diff
@@ -1,9 +1,13 @@
 admins = { "{{ .Env.JICOFO_AUTH_USER }}@{{ .Env.XMPP_AUTH_DOMAIN }}" }
 plugin_paths = { "/prosody-plugins/", "/prosody-plugins-custom" }
 http_default_host = "{{ .Env.XMPP_DOMAIN }}"
-
+ 
 {{ $ENABLE_AUTH := .Env.ENABLE_AUTH | default "0" | toBool }}
 {{ $AUTH_TYPE := .Env.AUTH_TYPE | default "internal" }}
+{{ $JWT_ASAP_KEYSERVER := .Env.JWT_ASAP_KEYSERVER | default "" }}
+{{ $JWT_ALL_EMPTY := .Env.JWT_ALLOW_EMPTY | default "0" | toBool }}
+{{ $JWT_AUTH_TYPE := .Env.JWT_AUTH_TYPE | default "token" }}
```

Comment:
why is this needed?

Reply:
The ASAP_KEYSERVER is used during ASAP authorization, where the public key is fetched from a keyserver.  The ALLOW_EMPTY is used to allow anonymous users when they have no JWT, while still validating incoming JWTs when provided.  The AUTH_TYPE is to control which JWT authorization module to use for incoming JWTs.